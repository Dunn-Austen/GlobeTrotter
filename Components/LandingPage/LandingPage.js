import React from 'react';
import { StyleSheet, Text, View, Image, TouchableOpacity } from 'react-native'

export const LandingPage = ({ navigation }) => {
  return (
    <View style={styles.container}>
      <View style={styles.globeContainer}>
        <Image 
          source={{ uri: 'https://i.imgur.com/ixGj1Nf.png' }} 
          style={styles.globe}
        />
      </View>
      <Text style={styles.title}>GlobeTrotter</Text>
      <TouchableOpacity
        activeOpacity={.8}
        style={styles.getStarted}
        onPress={() => navigation.navigate('Trips')}
      >
        <Text style={styles.buttonText}>Get Started</Text>
      </TouchableOpacity>
    </View>
  )
}

const styles = StyleSheet.create({
  container: {
    backgroundColor: '#1E88E5',
    height: '100%',
    alignItems: 'center',
    padding: 15,
    justifyContent: 'space-around'
  },
  getStarted: {
    backgroundColor: '#2CCB70',
    width: '100%',
    height: 50,
    borderRadius: 5,
    alignItems: 'center',
    justifyContent: 'center',
    marginTop: 15
  },
  title: {
    color: '#fff',
    fontSize: 50,
    fontWeight: 'bold'
  },
  buttonText: {
    color: '#0D47A1',
    fontWeight: 'bold',
    fontSize: 20
  },
  globe: {
    width: 300,
    height: 300,
    marginLeft: 20
  }
})
export default LandingPage;