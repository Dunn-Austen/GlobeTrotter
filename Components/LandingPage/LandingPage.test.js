import React from 'react';
import LandingPage from './LandingPage';

import renderer from 'react-test-renderer';

test.skip('renders correctly', () => {
  const tree = renderer.create(<LandingPage />).toJSON();
  expect(tree).toMatchSnapshot();
});