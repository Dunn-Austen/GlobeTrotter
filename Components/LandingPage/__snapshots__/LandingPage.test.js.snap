// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#1E88E5",
      "height": "100%",
      "justifyContent": "space-around",
      "padding": 15,
    }
  }
>
  <View>
    <Image
      source={
        Object {
          "uri": "https://i.imgur.com/ixGj1Nf.png",
        }
      }
      style={
        Object {
          "height": 300,
          "marginLeft": 20,
          "width": 300,
        }
      }
    />
  </View>
  <Text
    style={
      Object {
        "color": "#fff",
        "fontSize": 50,
        "fontWeight": "bold",
      }
    }
  >
    GlobeTrotter
  </Text>
  <View
    accessible={true}
    focusable={true}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#2CCB70",
        "borderRadius": 5,
        "height": 50,
        "justifyContent": "center",
        "marginTop": 15,
        "opacity": 1,
        "width": "100%",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#0D47A1",
          "fontSize": 20,
          "fontWeight": "bold",
        }
      }
    >
      Get Started
    </Text>
  </View>
</View>
`;
